src/main/java/com/google/error_analyzer/backend/LogDao.java

```diff
@@ -166,6 +168,15 @@ public String getJsonStringById (String fileName, String id)
         return getResponse.getSourceAsString();  
     }
 
+    //delete indices
+    @Override
+    public String deleteIndices(String sessionId) throws IOException {
+        String indexName = sessionId.concat("*");
```

Comment:
what if sessionID = ""?

Reply:
I changed the default sessionID to "invalidSession" so if the cookies is null (exception case) session id will be "invalidSession", if user clicks the delete button , then all the indices with index name pattern "indexName*" would be deleted. sessionID will not be equal to "" in any case.

Reply:
Verifying: Will all indices with index name pattern "invalidSession*" be deleted (and not "indexName*")?

Approach sounds good to me.

Reply:
Sorry, indices with index name pattern  "invalidSession*" will be deleted